### PR TITLE
Fixing some of the linting errors

### PR DESCRIPTION
**Pull request by CsatariGergely**
This is still in the works, do not approve it. I just need the Vale job to run.

### CODE REVIEW

**Review by s3wong — COMMENTED** [review]
/lgtm

**Comment by s3wong** [comment]
@CsatariGergely vale seems to complain on some capitalization (and couple on choice of wording) errors. PTAL.

**Comment by CsatariGergely** [comment]
> @CsatariGergely vale seems to complain on some capitalization (and couple on choice of wording) errors. PTAL.

Yep, it is a massive list :) 

I will also configure it not to fail the check, but only post warnings. I would like to use these as more like recommendations for now.

**Comment by CsatariGergely** [comment]
Now it is strange, that the Vale check is not triggered for every update.

File: content/en/docs/release-notes/R2.md [hunk 1]
```diff
@@ -129,11 +129,11 @@ The following limitations need to be borne in mind:
   does not match src line`.
   If such a message appears, then retry in a little while, as this error may resolve itself. Restarting
   Porch may also help.
-* During specialization, we may have duplicate parameterRef extensions, leading to failed deployments
+* During specialization, we may have duplicate parameterRef extensions, leading to unsuccesfull deployments
```
Comment (liamfallon) [inline content/en/docs/release-notes/R2.md:132]:
```suggestion
* During specialization, we may have duplicate parameterRef extensions, leading to unsuccesful deployments
```

Comment (liamfallon) [reply content/en/docs/release-notes/R2.md:132]: I don't know why the word "unsuccessful" is spelt that way in English (it makes no sense) :-D

Comment (s3wong) [reply content/en/docs/release-notes/R2.md:132]: Actually I don't know why our doc checking/lint tool would find "failed" deployments offensively, it is a much better wording to capture the reality than an "unsuccessful" deployment.

Comment (CsatariGergely) [reply content/en/docs/release-notes/R2.md:132]: Okay, I can change it back to failed, if you think that it is better in this context.

**Comment by CsatariGergely** [comment]
> Now it is strange, that the Vale check is not triggered for every update.

This is fixed now.

File: content/en/docs/release-notes/R2.md (hunk 1 above)
Comment (CsatariGergely) [reply content/en/docs/release-notes/R2.md]: This is changed back now.

**Comment by CsatariGergely** [comment]
/assign @arora-sagar @liamfallon 

This is ready for merge now.

**Pull request merged**